app/Http/Controllers/Legacy/Web/ActionsController.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace Rogue\Http\Controllers\Legacy\Web;
+
+use Rogue\Models\Action;
+use Illuminate\Http\Request;
+use Rogue\Http\Controllers\Controller;
+
+class ActionsController extends Controller
+{
+    /**
+     * Create a controller instance.
+     */
+    public function __construct()
+    {
+        $this->middleware('auth');
+        $this->middleware('role:admin,staff', ['only' => ['create', 'store', 'edit', 'update', 'destroy']]);
```

Comment:
I thought we didn't want to allow folks to delete actions through the web interface?

Reply:
I think we don't want to make it obvious but from what I understand, an action can be edited/deleted until the campaign is live and then after that, they can't edit or delete it. Is that right @ngjo ?